src/lib.rs

```diff
@@ -509,30 +591,43 @@ mod css_color_tests {
 
     #[test]
     fn can_convert_to_rgb_notations() {
-        let rgb_color = RGB::new(5, 10, 15);
-        let rgba_color = RGBA::new(5, 10, 15, 255);
-        let _hsl_color = HSL::new(6, 93, 71);
-        let _hsla_color = HSLA::new(6, 93, 71, 255);
+        let rgb_color = RGB::new(24, 98, 119);
+        let rgba_color = RGBA::new(24, 98, 119, 255);
+        let hsl_color = HSL::new(193, 67, 28);
+        let hsla_color = HSLA::new(193, 67, 28, 255);
+
+        // HSL to RGB
+        assert_eq!(hsl_color.to_rgb(), rgb_color);
```

Comment:
I've been using [this](http://www.niwa.nu/2013/05/math-behind-colorspace-conversions-rgb-hsl) version of the conversion algorithm, but something appears to be wrong. If you convert `HSL(193, 67, 28)` [here](https://www.rapidtables.com/convert/color/hsl-to-rgb.html), then you should actually get `RGB(24, 99, 119)`.  Something seems fishy 🤔

cc @chancancode